book/tutorials/gpr/gpr.ipynb

```diff
@@ -0,0 +1,1305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "saving-terrorism",
+   "metadata": {},
+   "source": [
+    "# SnowEx Hackweek GPR Tutorial 2021\n",
+    "\n",
+    "**Overview**\n",
+    "\n",
+    "In this tutorial we will request the snow pit 1S1 location and density, ground-penetrating radar (GPR) two-way travel-time (TWT) and geolocation data, and Magnaprobe snow depths and location to make a quick comparison of the Magnaprobe snow depth measurements and the GPR estimated snow depths.\n",
+    "\n",
+    "We will calculate the average density from the pit and visualize a set of GPR travel-times around Pit 1S1. Given the average dry snow density of 1S1, we will then use an empirically derived expression from Kovacs et. al (1995) to estimate the radar wave speed. The wave speed allows us to convert the radar two-way travel-time to snow depth.\n",
+    "\n",
+    "Lastly we will use a few summary statistics to compare the GPR and Magnaprobe snow depths, and we will discuss the various potential sources of error that arise naturally, systematically, and/or algorithmically.\n",
+    "\n",
+    "**Slides**\n",
+    "\n",
+    "https://docs.google.com/presentation/d/1Hh2CdCCvhWzcWzjzHi9WPum5NmOQt67B1vi1AwOzXCQ/edit?usp=sharing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "hybrid-warren",
+   "metadata": {},
+   "source": [
+    "# Retrieve density, GPR, and Magnaprobe data from Pit 1S1\n",
```

Comment:
You'll want to change this to `## Retrieve density, ` so that `# SnowEx Hackweek GPR Tutorial 2021` gets rendered as the tutorial title on the website

Reply:
Fixed!